rfcs/text/0428-extending-and-duplicating-types.md

```diff
@@ -776,6 +851,123 @@ The above change applies to these Block Protocol operations (`updateEntityType`
 
 which must all make use of complete schemas (absent of `$id`) in place of partial ones. These examples originate from the [Type System RFC](./0352-graph-type-system.md#interfacing-with-types-1).
 
+## Removing link ordering
+
+The [current meta schema for `Entity Type`](https://blockprotocol.org/types/modules/graph/0.4/schema/entity-type) allows a `ordered` property inside of `links` and [current meta schema for `Link Data`](https://blockprotocol.org/types/modules/graph/0.3/schema/link-data) includes the [link ordering schema](https://blockprotocol.org/types/modules/graph/0.3/schema/link-orders) as properties, which consists of `leftToRightOrder` and `rightToLeftOrder`. With the removal of link ordering, these will be removed in the next version of the meta schema, so the next version of the `Entity Type` meta schema will look like this (the changes to the `allOf` property from above were omitted):
```

Comment:
Just a few nits, the link data and link orders are only schemas, they're not metaschemas (as they don't describe a schema):

```suggestion
The [current metaschema for `Entity Type`](https://blockprotocol.org/types/modules/graph/0.4/schema/entity-type) allows an `ordered` property inside of `links` and [current schema for `Link Data`](https://blockprotocol.org/types/modules/graph/0.3/schema/link-data) includes the [link ordering schema](https://blockprotocol.org/types/modules/graph/0.3/schema/link-orders) as properties, which consists of `leftToRightOrder` and `rightToLeftOrder`. With the removal of link ordering, these will be removed in the next version of the metaschema, so the next version of the `Entity Type` metaschema will look like this (the changes to the `allOf` property from above were omitted):
```